RobotCode/src/main/java/IntakeSubsystem.java

```diff
@@ -0,0 +1,49 @@
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import frc.lib.core.TeamSubsystemBase;
+import frc.robot.constants.Ports;
+import frc.lib.core.motors.TeamSparkMAX;
+
+public class IntakeSubsystem extends TeamSubsystemBase
+{
+    private DoubleSolenoid m_doubleSolenoid;
+    private TeamSparkMAX intakeMotor;
+    private double motorSpeed;
+
+    public IntakeSubsystem(int forwardChannel, int reverseChannel)
+    {
+        m_doubleSolenoid = new DoubleSolenoid(PneumaticsModuleType.REVPH, 0, 0);
+        intakeMotor = new TeamSparkMAX("Intake Motor", Ports.INTAKE_MOTOR);
+        motorSpeed = 0.5;
+        m_doubleSolenoid.set(DoubleSolenoid.Value.kReverse);
+
+    }
+
+    public String toggleIntake()
```

Comment:
Don't use Strings for returns like this. You can do Optional<DoubleSolenoid.Value> and return DoubleSolenoid.Value.kForward, DoubleSolenoid.Value.kReverse, or null